pyproject.toml

```diff
@@ -47,14 +47,29 @@ dev = [
     "ruff==0.5.2",
     "bandit==1.7.9",
     "types-requests==2.32.0.20240622",
-    "gradio==4.37.2"
+    "gradio==4.37.2",
+    "boto3==1.34.145",
 ]
 
 [[tool.pdm.source]]
 type = "find_links"
 url = "https://download.pytorch.org/whl/cpu/torch_stable.html"
 name = "torch"
 
+
```

Comment:
why it was moved?

Reply:
Don't know. when I run pdm self update, it moves this.
I can move it back, no problem

Reply:
please do, let's keep the changes minimal. TY